shpkpr/template_filters.py

```diff
@@ -8,10 +8,23 @@ def filter_items(value, startswith=None, strip_prefix=False):
 
     Returns a list of key/value tuples.
 
-    Usage:
+    .. code-block:: jinja
+
         {{ my_dict|filter_items }}
         {{ my_dict|filter_items("MY_PREFIX_") }}
         {{ my_dict|filter_items("MY_PREFIX_", True) }}
+
+    This is most useful in combination with the special
+    :ref:`_all_env <all_env>` variable that shpkpr injects into every template.
+    For example, to iterate over only the template variables that start with
+    ``LABEL_`` you could do:
```

Comment:
I don't love that this leaks some restructuredtextisms/sphinxisms into the docstrings, but on the other hand we do get pretty HTML docs because of it :man_shrugging: